docs/How-to-test-a-PR.md

```diff
@@ -112,6 +120,18 @@ You should write:
 - please insert as many details as possible
 - you can also insert screenshot or videos
 
+## Get Back to Master
+
+Once you have finished testing you likely want to return to the master branch.
+
+To do this you must run;
+
+```git checkout master```
+
+```git pull```
+
+```./bin/acore-docker-build```
```

Comment:
here is better to just say "rebuild" as this way of rebuilding is specific for the Docker setup